xc/cache.lua

```diff
@@ -35,9 +35,10 @@ end
 -- @return true if authorized, false if denied, nil if unknown
 -- @return reason why the authorization is denied (optional)
 function _M.authorize(service_id, credentials, usage_method)
-  local redis, ok = redis_pool.acquire()
+  local redis, ok, err = redis_pool.acquire()
 
   if not ok then
+    ngx.log(ngx.WARN, "[cache] couldn't connect to redis: ", err)
```

Comment:
If we do, we should change this error message to a more specific one, so the user knows whether the error happened while authorizing or while reporting.